example/test_problem/ELBDM/RestrictionMismatch/Input__Parameter_PhaseResOnLongRun

```diff
@@ -0,0 +1,175 @@
+
+
+# =================================================================================================================
+# NOTE:
+# 1. Comment symbol: #
+# 2. [*]: defaults
+# 3. Parameters set to "auto" (usually by setting to a negative value) do not have deterministic default values
+#    and will be set according to the adopted compilation options and/or other runtime parameters
+# 4. To add new parameters, please edit "Init/Init_Load_Parameter.cpp"
+# 5. All dimensional variables should be set consistently with the code units (set by UNIT_L/M/T/V/D) unless
+#    otherwise specified (e.g., SF_CREATE_STAR_MIN_GAS_DENS & SF_CREATE_STAR_MIN_STAR_MASS)
+# 6. For boolean options: 0/1 -> off/on
+# =================================================================================================================
+
+
+# simulation scale
+BOX_SIZE                      8.0        # box size along the longest side (in Mpc/h if COMOVING is adopted)
+NX0_TOT_X                     128        # number of base-level cells along x
+NX0_TOT_Y                     128        # number of base-level cells along y
+NX0_TOT_Z                     16         # number of base-level cells along z
+OMP_NTHREAD                   2          # number of OpenMP threads (<=0=auto -> omp_get_max_threads) [-1] ##OPENMP ONLY##
```

Comment:
Not sure whether it will be better to use `OMP_NTHREAD 1` to make all the test runs consistent, or we maintain `OMP_NTHREAD 2` to accelerate the simulation~